src/core/status.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+// Possible IPFS status values:
+module.exports = {
+  new: 'new',
+  initializing: 'initializing',
+  initialized: 'initialized',
+  starting: 'starting',
+  running: 'running',
+  stopping: 'stopping',
+  stopped: 'stopped'
+}
```

Comment:
Rechecked, this is just used to populate a var `this._status` once and that const is not used anymore. ???